src/subapps/studioLegacy/containers/WorkspaceMenuContainer.tsx

```diff
@@ -484,22 +485,23 @@ const WorkspaceMenu: React.FC<WorkspaceMenuProps> = ({
     data: TableResource | UnsavedTableResource
   ) => {
     if (selectedDashboard) {
-      const resource = await nexus.Resource.get(
+      const resource = (await nexus.Resource.get(
         orgLabel,
         projectLabel,
         encodeURIComponent(selectedDashboard['@id'])
-      );
+      )) as StudioResource;
       await nexus.Resource.update(
         orgLabel,
         projectLabel,
         encodeURIComponent(selectedDashboard['@id']),
         selectedDashboard._rev,
         {
-          ...resource,
+          ...resourceWithoutMetadata(resource),
           description: data.description,
           label: data['name'],
         }
       ).catch(err => {
+        console.log('Error here');
```

Comment:
I am not sure if this was intended to stay, but if so, seems a bit too general as message

Reply:
:laughing: Nice catch. It's a mistake I should clean up. Thanks @crisely09